docs/sources/datasources/grafana-pyroscope.md

```diff
@@ -30,67 +30,86 @@ To configure basic settings for the data source, complete the following steps:
-   | `User`         | User name for basic authentication.                                                                                                                                                                                                                                                                                      |
-   | `Password`     | Password for basic authentication.                                                                                                                                                                                                                                                                                       |
-   | `Minimal step` | Used for queries returning timeseries data. Phlare backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval use Minimal step same or higher than your Phlare scrape interval. For Pyroscope backend this prevents returning too many data points to the front end. |
+   | Name           | Description                                                                                                                                                                                                                                                                                                               |
+   | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | `Name`         | A name to specify the data source in panels, queries, and Explore.                                                                                                                                                                                                                                                        |
+   | `Default`      | The default data source will be pre-selected for new panels.                                                                                                                                                                                                                                                              |
+   | `URL`          | The URL of the Grafana Pyroscope instance, for example, `http://localhost:4100`                                                                                                                                                                                                                                        |
```

Comment:
```suggestion
   | `URL`          | The URL of the Grafana Pyroscope instance, for example, `http://localhost:4100`.                                                                                                                                                                                                                                       |
```

Or does it add confusion if there's a full-stop here?

Reply:
Nope. The other rows in the tables have periods, so I think that is good